### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by azeghers**
Updating the CHANGELOG.md in preparation for bpmn-js-properties-panel@0.39.0

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,16 @@ ___Note:__ Yet to be released changes appear here._
...
 
+## 0.39.0
+
+* `FEAT`: add 'Participant' prefix to properties panel participant inputs for clarity ([#413](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/413))
+* `CHORE`: bump to `bpmn-js@8.1.0`
+* `CHORE`: bump to `diagram-js@7.0.0`
+
+### BREAKING CHANGES
+
+* Participant input `data-entry` html attribute is now `participant-id` from `id`
```
Comment (MaxTru) [inline CHANGELOG.md:19]: _~~from~~ was `id`_

**Review by MaxTru — COMMENTED** [review]
LGTM

Only one minor minor suggestion

File: CHANGELOG.md (hunk 1 above)
Comment (azeghers) [reply CHANGELOG.md:19]: I think it's correct but I see how it can sound uncanny after reading again.
How would you phrase it?

Comment (MaxTru) [reply CHANGELOG.md:19]: Participant input `data-entry` html attribute was changed from `id` to `participant-id`

**Review by MaxTru — APPROVED** [review]

**Pull request merged**